proto/binlog/binlog.proto

```diff
@@ -10,16 +10,20 @@ option (gogoproto.unmarshaler_all) = true;
 
 // TableMutation contains mutations in a table.
 message TableMutation {
-    optional int64 table_id     = 1 [(gogoproto.nullable) = false];
-    repeated int64 inserted_ids = 2;
-    repeated int64 updated_ids  = 3;
-    repeated int64 deleted_ids  = 4;
+    optional int64 table_id      = 1 [(gogoproto.nullable) = false];
 
-    // For tables with non-interger types primary key.
-    repeated bytes deleted_pks = 5;
+    // For inserted rows and updated rows, we save all column values of the row.
+    repeated bytes inserted_rows = 2;
```

Comment:
It's going to be saved in local file, saving the row values so we don't need to look up the row in the TiKV.